ground/gcs/src/plugins/boards_aeroquad/aq32.cpp

```diff
@@ -184,8 +181,6 @@ Core::IBoardType::InputType AQ32::getInputType()
     switch(settings.RcvrPort) {
     case HwAQ32::RCVRPORT_PPM:
         return INPUT_TYPE_PPM;
-    case HwAQ32::RCVRPORT_PWM:
-        return INPUT_TYPE_PWM;
     default:
         break;
     }
```

Comment:
Yep. I prefer to have an explicit list without a default case, so if anything new is added, you are forced to review it rather than silently breaking things or missing a board out (this has happened a lot in the past).